cassandra/pool.py

```diff
@@ -926,4 +928,6 @@ def get_connections(self):
 
     def get_state(self):
         in_flights = [c.in_flight for c in self._connections]
-        return {'shutdown': self.is_shutdown, 'open_count': self.open_count, 'in_flights': in_flights}
+        orphan_requests = [c.orphaned_request_ids for c in self._connections]
+        return {'shutdown': self.is_shutdown, 'open_count': self.open_count, \
+            'in_flights': in_flights, 'orphan_requests': orphan_requests}
```

Comment:
These changes were necessary to expose orphan request IDs to the test infrastructure easily.  Fortunately they also make sense generally.